src/audio/drc/testbench_drc_asin/drc_asin_fixed.c

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
```

Comment:
in my option this file should still go under tools/ and we should use cmake to access the files through the directories. @lgirdwood thoughts?

Reply:
Ack, best to go in tools.